src/AudioUnit/AudioComponent.cs

```diff
@@ -448,6 +448,10 @@ public static class AudioComponentValidationParameter {
 //		#define kAudioComponentValidationParameter_ForceValidation		 "ForceValidation"
 		public static NSString ForceValidation = new NSString ("ForceValidation");
 
+//		#define kAudioComponentValidationParameter_LoadOutOfProcess		 "LoadOutOfProcess"
+		[iOS (14,5), Mac (11,3)]
+		public static NSString LoadOutOfProcess = new NSString ("LoadOutOfProcess");
```

Comment:
it's added in the tvOS assembly, should it be ?
if so there should be an availability attribute